src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/model/FeedItem.java

```diff
@@ -0,0 +1,16 @@
+package org.getlantern.lantern.model;
+
+public class FeedItem {
+    public String Title;
```

Comment:
Fixed. The reason the variables were uppercase and public before is because I was previously using a library that necessitated it to parse JSON into FeedItem objects.